static/_redirects

```diff
@@ -183,6 +183,10 @@
 /guides/launch/run-agent/ /guides/launch/setup-agent-advanced/ 301
 /guides/launch/sagemaker/ /guides/launch/setup-launch-sagemaker/ 301
```

Comment:
@johndmulhausen Should "guides" be in the dev guide Launch redirect?

Reply:
Update: I believe these are in place because the `url` keys in the respective pages contain "guides/launch/...."